### PR TITLE
core/tracker: update tracker for DutyAggregator

### PR DESCRIPTION
**Pull request by dB2510**
Adds support for DutyAggregator and DutyPrepareAggregator failures to tracker.

category: feature
ticket: #1154

### CODE REVIEW

File: core/fetcher/fetcher.go [hunk 1]
```diff
@@ -77,6 +77,8 @@ func (f *Fetcher) Fetch(ctx context.Context, duty core.Duty, defSet core.DutyDef
 		unsignedSet, err = f.fetchAggregatorData(ctx, duty.Slot, defSet)
 		if err != nil {
 			return errors.Wrap(err, "fetch aggregator data")
+		} else if len(unsignedSet) == 0 { // No aggregators found in this slot
```
Comment (corverroos) [inline core/fetcher/fetcher.go:80]: should add a test for this

Comment (corverroos) [inline core/fetcher/fetcher.go:80]: we need to add tests for this.

**Review by corverroos — APPROVED** [review]

File: core/tracker/tracker.go [hunk 2]
```diff
@@ -33,28 +33,33 @@ type component int
...
+	fetcherAggregatorZeroPreparesMsg  = "couldn't aggregate attestation due to zero partial committee subscriptions"
+	fetcherAggregatorFailedPrepareMsg = "couldn't aggregate attestation due to failed prepare aggregator duty "
+	fetcherProposerFewRandaosMsg      = "couldn't propose block due to insufficient partial randao signatures"
+	fetcherProposerZeroRandaosMsg     = "couldn't propose block due to zero partial randao signatures"
+	fetcherProposerFailedRandaoMsg    = "couldn't propose block due to failed randao duty"
+	consensusMsg                      = "consensus algorithm didn't complete"
+	validatorAPIMsg                   = "signed duty not submitted by local validator client"
+	parSigDBInternalMsg               = "partial signature database didn't trigger partial signature exchange"
+	parSigExMsg                       = "no partial signatures received from peers"
+	parSigDBThresholdMsg              = "insufficient partial signatures received, minimum required threshold not reached"
```
Comment (OisinKyne) [inline core/tracker/tracker.go:62]: @dB2510 @thomasheremans we should consider a collapsed section of the FAQ on the doc site that explains each of these errors in two sentences or so. We should aim that if people google these errors exactly our docs show up in the indexed google results.

Comment (thomasheremans) [reply core/tracker/tracker.go:62]: Sure @dB2510 can you provide me with the two sentences or so for each and I will add them to the docs

File: core/fetcher/fetcher.go (hunk 1 above)
Comment (corverroos) [reply core/fetcher/fetcher.go:80]: done

File: core/tracker/tracker.go (hunk 2 above)
Comment (thomasheremans) [reply core/tracker/tracker.go]: Thanks @corverroos , @OisinKyne this has been added to the PR https://github.com/ObolNetwork/obol-docs/compare/main...Launchpad-guide-and-faq

**Pull request merged**